step.yml

```diff
@@ -49,39 +21,49 @@ is_always_run: false
 is_skippable: false
 run_if: ""
 
-deps:
-  brew:
-  - name: git
-  - name: wget
-  apt_get:
-  - name: git
-  - name: wget
-
 
 toolkit:
   bash:
     entry_file: step.sh
 
 
 inputs:
-  - example_step_input: Default Value - you can leave this empty if you want to
+  - repository:
     opts:
-      title: "Example Step Input"
-      summary: Summary. No more than 2-3 sentences.
-      description: |
-        Description of this input.
-
-        Can be Markdown formatted text.
+      title: GitHub repository
+      summary: The full name of the repository, in the format {owner}/{repo}, e.g. deliveroo/mobile-scripts.
       is_expand: true
       is_required: true
-      value_options: []
-
-outputs:
-  - EXAMPLE_STEP_OUTPUT:
+  - script_path:
+    opts:
+      title: The path of the script inside the repo
+      summary: The path is relative to the root of the repo.
+      is_expand: true
+      is_required: true
+  - branch: "master"
+    opts:
+      title: The git branch
+      summary: The default value is "master" but if your script leaves in a specific git branch, you need to specify it.
+      is_expand: true
+      is_required: true
+  - command_line_arguments:
+    opts:
+      title: The arguments to pass to the remote script
+      summary: Each script has its own usage and expects a completely different set of command line arguments. E.g. if a script usage says `script.sh --key1 val1 --key2 val2`, just write `--key1 val1 --key2 val2`, omitting the name of the script.
+      is_expand: true
+      is_required: true
```

Comment:
Do we need to support scripts without arguments?

Reply:
Good point. I set `is_required` to `false` and also I tested it by removing the `command_line_arguments` input in the `cli-test` (with the `test-script.sh`) and it's still working correctly 💪